java/server/world/entity/mob/player.mcdoc

```diff
@@ -54,9 +54,9 @@ dispatch minecraft:entity[player] to struct Player {
 	XpTotal: int,
 	/// Seed for enchantments.
 	XpSeed: int,
-	Inventory: [SlottedItem] @ 0..41,
+	Inventory: [SlottedItem<255>] @ 0..41,
```

Comment:
I think the max value is 103 for the head slot here. And yeah slot ids 36-98 don't exist, which I guess ideally would be caught, but I suppose it would require some sort of hack

Reply:
You could use `SlottedItem<byte @ 0..35 | byte @ 100..103 | byte @ -106>`